src/public/outlines/outline_polygon.js

```diff
@@ -0,0 +1,15 @@
+var OutlinePolygon = function(outerRing, innerRings) {
```

Comment:
As commented above, suggest not creating new types for OutlinePolygon / OutlinePolygonRing - represent in the same way that we already use for L.Wrld.Polygon instead